packages/ui-components/src/components/icon/icon.types.ts

```diff
@@ -130,6 +133,8 @@ export enum EIconName {
 	SortZa = 'kv-sort-za',
 	Square = 'kv-square',
 	Status = 'kv-status',
+	StarActive = 'kv-star-active',
+	StarInactive = 'kv-star-inactive',
```

Comment:
the icon is the same with different opacity right? that opacity can be added by css you don't need to have different icons

Reply:
Yes, but is used as an icon. I mean, in every single place where you want to use an inactive start you must pass this opacity setting. That's why I choose to make it an icon.

Reply:
that way we should have an inactive state on all icons. If you really need this feature, I suggest adding a property to `kv-icon` to set the inactive state.

Reply:
I see your point, but we cannot be sure that all inactive icons will all the same state.

In this case, I'll remove the inactive icon svg and style it as need 👍